src/thunderbots/software/ai/hl/stp/evaluation/enemy_threat.cpp

```diff
@@ -157,3 +159,125 @@ std::optional<std::pair<int, std::optional<Robot>>> Evaluation::getNumPassesToRo
     // state. Therefore, we return an std::nullopt
     return std::nullopt;
 }
+
+std::vector<Evaluation::EnemyThreat> Evaluation::sortThreatsInDecreasingOrder(
+    std::vector<Evaluation::EnemyThreat> threats)
```

Comment:
Hmm, thoughts on making this sort in-place? That's kinda the standard as far as sorting in C++ goes?

Reply:
No strong preference, was just sticking with our more functional idiom. I'll change it to in-place

Reply:
Looks like this isn't changed?